tests/unit/test_views_posts.py

```
@@ -154,9 +154,7 @@ def test_update_own_post_returns_200(self, auth_client, post):
 
     def test_update_other_user_post_returns_403(self, api_client, post, db):
         """A user cannot edit another user's post."""
-        other = User.objects.create_user(
-            username="other", email="other@x.com", password="p"
-        )
+        other = User.objects.create_user(username="other", email="other@x.com", password="p")
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "password"

(S106)